docs/articles/nunit/writing-tests/TestContext.md

```diff
@@ -143,9 +148,25 @@ Gets a representation of the current test, with the following properties:
 
 * **ID** - The unique Id of the test
 * **Name** - The name of the test, whether set by the user or generated automatically
+* **Namespace** - The namespace of the test
+* **DisplayName** - The display name of the test
 * **FullName** - The fully qualified name of the test
 * **MethodName** - The name of the method representing the test, if any
-* **Properties** - An `IPropertyBag` of the test properties
+* **Method** - The IMethodInfo of the test method
+* **Type** - The Typeinfo of the test
```

Comment:
```suggestion
* **Type** - The `TypeInfo` of the test
```

I think this is correct? (if so, it'll let you remove the cSpell dictionary addition as well I think